package.json

```diff
@@ -71,14 +71,14 @@
     "rimraf": "^3.0.2",
     "rollup": "^2.23.0",
     "rollup-plugin-cleanup": "^3.1.1",
-    "standardx": "^5.0.0",
+    "standardx": "^7.0.0",
```

Comment:
mind making the `standardx` upgrade in a separate PR?

Reply:
sure